walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -58,8 +55,19 @@ public void setUp() {
     @Test
     public void addJob_NormalInput_success() throws ExecutionException, InterruptedException {
         // Arrange.
+        JobStatus expectedJobStatus = JobStatus.ACTIVE;
         String expectedJobName = "Software Engineer";
-        Job job = new Job(expectedJobName);
+        Location expectedLocation =  new Location("Google", 0, 0);
+        String expectedJobDescription = "Programming using java";
+        JobPayment expectedJobPayment = new JobPayment(0, 5000, Frequency.MONTHLY);
+        List<String> expectedRequirements = Arrays.asList("O Level", "Driving License");
```

Comment:
I think this is a list of the requirement ids e.g., "o-levels"

Reply:
Also note that this would need to be changed in the below tests too

Reply:
Fixed